UI/GameSettingsScreen.cpp

```diff
@@ -1069,7 +1069,7 @@ void RecreateActivity() {
 }
 
 UI::EventReturn GameSettingsScreen::OnAdhocGuides(UI::EventParams &e) {
-	LaunchBrowser("https://forums.ppsspp.org/forumdisplay.php?fid=34");
+	LaunchBrowser("https://github.com/hrydgard/ppsspp/wiki/How-to-play-multiplayer-games-over-PPSSPP");
```

Comment:
The landing page link may need to go through translation too isn't? if we wanted to support multiple wiki pages with different language each?

Reply:
I guess so but I dont know where the translation stuff is.

Reply:
That's a good point - and probably a good idea! Should name it with a second argument then. Like:

`x->T("MultiplayerLink", "http://....")`

Reply:
> if we wanted to use multiple wiki pages with different language each?
I think so.

Reply:
So should I do it LaunchBrowser(x->T("MultiplayerLink","https://github.com/hrydgard/ppsspp/wiki/How-to-play-multiplayer-games-over-PPSSPP")); ?